.github/pull_request_template.md

```diff
@@ -43,5 +44,5 @@ Tick one of the following boxes [x] to help us understand if the base branch for
   Maintainers are happy to point out inconsistencies but in order to speed up the review and merge process we ask you to check the CODING standards.
 -->
 ## PR quality check
-- [ ] I checked the [CODING_STANDARDS.md](https://github.com/MariaDB/server/blob/11.0/CODING_STANDARDS.md) file and my PR conforms to this where appropriate.
+- [ ] I checked the [CODING_STANDARDS.md](https://github.com/MariaDB/server/blob/-/CODING_STANDARDS.md) file and my PR conforms to this where appropriate.
```

Comment:
The link actually works and leads to the current main branch. Nice!

Reply:
Ooo... I was wondering how to do that, I like it :)